common/schemas/kpj/kpj.schema.json

```
@@ -0,0 +1,124 @@
+{
+  "title": "kpj.xsd",
+  "$schema": "http://json-schema.org/schema#",
+  "type": "object",
+  "properties": {
+    "KeymanDeveloperProject": {
+      "properties": {
+        "Options": {
+          "$ref": "#/definitions/Options"
+        },
+        "Files": {
+          "$ref": "#/definitions/Files"
+        }
+      },
+      "required": [
+        "Options",
+        "Files"
+      ],
+      "additionalProperties": false,
+      "type": "object"
+    }
+  },
+  "required": [
+    "KeymanDeveloperProject"
+  ],
+  "additionalProperties": false,
+  "definitions": {
+    "Options": {
+      "type": "object",
+      "properties": {
+        "BuildPath": {
+          "type": "string"
+        },
+        "CompilerWarningsAsErrors": {
+          "type": "string",
+          "pattern": "^(True|False)$"
+        },
+        "WarnDeprecatedCode": {
+          "type": "string",
+          "pattern": "^(True|False)$"
+        },
+        "CheckFilenameConventions": {
+          "type": "string",
+          "pattern": "^(True|False)$"
+        },
+        "ProjectType": {
+          "type": "string",
+          "pattern": "^(keyboard|lexicalmodel)$"
```

Is this `KeymanDeveloperProjectType` in keyman-developer-project.ts?
```suggestion
          "pattern": "^(Keyboard|LexicalModel)$"
```

---

No, that is an enum type in TypeScript, this is the XML and should be lower case.